geode-core/src/test/java/org/apache/geode/internal/cache/PoolManagerTest.java

```diff
@@ -37,6 +65,112 @@ public void setUp() {
     assertThat(poolManager.getMap()).isEmpty();
   }
 
+  // in Mulituser Auth mode
+  @Test
+  public void Test_thatDataSerializerSynchronizationMessagesAreNotSent() {
```

Comment:
You should use camel case. The method names should start with lowercase unless there's some unusual reason why it helps with organization and understanding of a class.